draft-ietf-jsonpath-base.md

```diff
@@ -1226,7 +1226,11 @@ The `filter-selector` works with arrays and objects exclusively. Its result is a
 A relative path, beginning with `@`, refers to the current array element or member value as the
 filter selector iterates over the array or object.
 
-Comparisons using one of the operators `<`, `<=`, `>`, and `>=` are between numeric values only.
+Comparison using one of the operators `==` or `!=` are between primitive values or singular paths. A singular path that selects no nodes compares unequal to a primitive value or a singular path which selects a node and compares
+equal to a singular path that selects no nodes. Two singular paths that each select a node compare equal if and only
+if these nodes hold equal primitive values.
+
+Comparisons using one of the operators `<`, `<=`, `>`, or `>=` are between numeric values only.
```

Comment:
Do we not want to include strings here?  (Note that we would need to define some kind of character ordering, e.g. ordinal, case-insensitive, etc.)

Reply:
@gregsdennis  I think we agreed to avoid "ordering comparisons" of strings. Anyway, this is existing text, so if you feel strongly, it deserves a separate issue.